src/styles/blocks/card-block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
```

Comment:
The 'align-items: center;' is missing. Add it to center the stars vertically within the flex container.